### ISSUE TITLE
Close one tab on the left/right

### ISSUE THREAD

**Issue by dyckia** [philc/vimium#3518]
Is there a way to close just one tab on the current tab's left/right? 
I only find the closeTabsOnLeft command which closes all tabs on the left.

**Comment by lazycatme** [comment]
map d closeTabsOnRight $count=1
map a closeTabsOnLeft $count=-1
这样?

**Comment by dyckia** [comment]
> map d closeTabsOnRight $count=1
> map a closeTabsOnLeft $count=-1
> 这样?

Just tested, but still closes all tabs on the left/right.

**Comment by gdh1995** [comment]
No, `$count` only exists on [Vimium C](https://github.com/gdh1995/vimium-c), which is a customized version of Vimium.
